### PR TITLE
[risk=no] RW-1763 trimming cdr verison table

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Removed unused columns from cdr version table.

### CODE REVIEW

File: common-api/src/main/java/org/pmiops/workbench/db/model/CdrVersion.java [hunk 1]
```diff
@@ -20,12 +20,8 @@
   private boolean isDefault;
   private String name;
   private Short dataAccessLevel;
```
Comment (calbach) [inline common-api/src/main/java/org/pmiops/workbench/db/model/CdrVersion.java]: also don't need

File: public-api/db/changelog/db.changelog-master.xml [hunk 2]
```diff
@@ -9,7 +9,8 @@
     <runningAs username="liquibase"/>
   </preConditions>
   <include file="changelog/db.changelog-1.xml"/>
+  <include file="changelog/db.changelog-2.xml"/>
```
Comment (calbach) [inline public-api/db/changelog/db.changelog-master.xml]: The environments all already exist so it's probably too late to be changing the migration files around, aside from renaming files. I suppose we could delete the databases if we really wanted to... Did you have a plan for this?

File: public-api/src/main/java/org/pmiops/workbench/publicapi/CdrVersionsController.java [hunk 3]
```diff
@@ -13,7 +13,6 @@
 import org.pmiops.workbench.db.model.CommonStorageEnums;
```
Comment (calbach) [inline public-api/src/main/java/org/pmiops/workbench/publicapi/CdrVersionsController.java]: Could you also remove this enum from the Swagger / enum converter utility?

**Review by calbach — COMMENTED** [review]
Sorry - one last thing I noticed. Otherwise looks good

File: public-api/src/main/java/org/pmiops/workbench/publicapi/CdrVersionsController.java (hunk 3 above)
Comment (SrushtiGangireddy) [reply public-api/src/main/java/org/pmiops/workbench/publicapi/CdrVersionsController.java]: Done. Thanks.

Comment (calbach) [reply public-api/src/main/java/org/pmiops/workbench/publicapi/CdrVersionsController.java]: So I actually was asking whether you could remove these methods entirely:

- https://github.com/all-of-us/data-browser/blob/master/common-api/src/main/java/org/pmiops/workbench/db/model/CommonStorageEnums.java#L51
- (field + enum): https://github.com/all-of-us/data-browser/blob/master/common-api/src/main/resources/common_api.yaml#L65

Comment (SrushtiGangireddy) [reply public-api/src/main/java/org/pmiops/workbench/publicapi/CdrVersionsController.java]: Was distracted working on something else. Pushed only the change from public-api.
Sorry.

File: common-api/src/main/resources/common_api.yaml [hunk 4]
```diff
@@ -62,7 +62,6 @@ definitions:
     required:
       - cdrVersionId
       - name
-      - dataAccessLevel
```
Comment (calbach) [inline common-api/src/main/resources/common_api.yaml:65]: Please remove the property and the enum and any other references in this file

File: common-api/src/main/resources/common_api.yaml [hunk 5]
```diff
@@ -50,6 +45,7 @@ definitions:
       - defaultCdrVersionId
     properties:
       items:
+        type: "array"
```
Comment (calbach) [inline common-api/src/main/resources/common_api.yaml]: revert?

**Review by calbach — APPROVED** [review]

**Pull request merged**